src/test/skript/tests/syntaxes/expressions/ExprBlockData.sk

```diff
@@ -1,4 +1,4 @@
-test "block data" when running minecraft "1.15.2":
+test "block data" when running minecraft "1.16.5":
```

Comment:
I don't know what to tell you. It says class doesn't exist.

My reasoning behind changing this, was because it's probably actually grabbing the correct type for the block data now.